### PR TITLE
RS+RT/YJ/RULE 22-14

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
I'd appreciate it if you could review this PR.

### CODE REVIEW

File: rct229/rules/section22/section22rule14.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        baseline_system_types = {
+            "SYS-7": ["hvac_sys_7"],
+            "SYS-11B": ["hvac_sys_11_b"],
+        }
+        # if any system type found in the APPLICABLE_SYS_TYPES then return applicable.
+        return any(
+            [key in APPLICABLE_SYS_TYPES for key in baseline_system_types.keys()]
+        )
+
+    class ChillerHeatRejectionRule(RuleDefinitionBase):
```
Comment (charliepnnl) [inline rct229/rules/section22/section22rule14.py:51]: This should probably be called `HeatRejectionRule` instead of `ChillerHeatRejectionRule`.

**Review by charliepnnl — APPROVED** [review]
Reviewed section22rule14.py

Approved with one comment to be considered.

File: rct229/rules/section22/section22rule14.py (hunk 1 above)
Comment (yunjoonjung-PNNL) [reply rct229/rules/section22/section22rule14.py]: Fixed. Great catch!

**Review by weilixu — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**